src/Core/Components/Forms/FluentValidationSummary.razor

```diff
@@ -0,0 +1,21 @@
+﻿@namespace Microsoft.Fast.Components.FluentUI
+@using Microsoft.AspNetCore.Components.Forms
+@inherits ValidationSummary
+
+@{ var validationMessages = Model is null ? FluentEditContext.GetValidationMessages() : FluentEditContext.GetValidationMessages(new FieldIdentifier(Model, string.Empty));
+
+    @if (validationMessages.Count() > 0)
+    {
+        <div @attributes="AdditionalAttributes">
+             <ul class="validation-errors">
+
+            @foreach (var error in validationMessages)
```

Comment:
It would be nice to use a `RenderFragments<T>` for all `validationMessages` elements.

Reply:
Agree. But we will work on that for a later version. As of now, each `validationMessage` is just a `string`